_docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/system/html.html.twig

```diff
@@ -97,74 +105,78 @@
                             </div>
                             <div class="sub-nav-group">
-                                <a class="heading"
-                                   href="/products#jboss_development_and_management">Middleware</a><a
-                                    class="hide-on-mobile" href="/products/devstudio/overview">Red
-                                Hat JBoss Developer Studio</a><a class="hide-on-mobile"
-                                                                 href="/products/eap/overview">Red
-                                Hat JBoss Enterprise Application Platform</a><a class="hide-on-mobile"
-                                                                                href="/products/fuse/overview">Red
-                                Hat JBoss Fuse</a><a class="hide-on-mobile"
-                                                     href="/products/webserver/overview">Red Hat
-                                JBoss Web Server</a><a class="hide-on-mobile"
-                                                       href="/products/bpmsuite/overview">Red Hat
-                                JBoss BPM Suite</a><a class="hide-on-mobile"
-                                                      href="/products/brms/overview">Red Hat JBoss
-                                BRMS</a><a class="hide-on-mobile"
-                                           href="/products/datagrid/overview">Red Hat JBoss Data
-                                Grid</a><a class="hide-on-mobile" href="/products/amq/overview">Red
-                                Hat JBoss A-MQ</a><a class="hide-on-mobile"
-                                                     href="/products/datavirt/overview">Red Hat
-                                JBoss Data Virtualization</a>
+                              <a class="heading" href="/products#integration_and_automation">Integration and Automation</a>
+                              <a class="hide-on-mobile" href="/products/amq/overview">Red Hat JBoss A-MQ</a>
+                              <a class="hide-on-mobile" href="/products/datavirt/overview">Red Hat JBoss Data Virtualization</a>
+                              <a class="hide-on-mobile" href="/products/fuse/overview">Red Hat JBoss Fuse</a>
+                              <a class="hide-on-mobile" href="/products/bpmsuite/overview">Red Hat JBoss BPM Suite</a>
+                              <a class="hide-on-mobile" href="/products/brms/overview">Red Hat JBoss BRMS</a>
                             </div>
+                          <div class="sub-nav-group">
+                            <a class="heading" href="/products#mobile">Mobile</a>
+                            <a class="hide-on-mobile" href="/products/mobileplatform/overview">Red Hat Mobile Application Platform</a>
+                          </div>
+                          <div class="sub-nav-group">
+                            <a class="heading" href="/products#cloud_products">Cloud</a>
```

Comment:
This isn't the same as production right now. In production it's called "Private Cloud" and the link should go to "/products#private_cloud.